Example/Shared/CustomStyleView.swift

```diff
@@ -49,9 +49,9 @@ struct CustomStyleView: View {
                endPoint: .bottomTrailing
            )
            .opacity(0.2)
-           .rotationEffect(selection % 2 == 0 ? Angle(degrees: 0) : Angle(degrees: 180))
+           .rotationEffect([.orange, .green, .purple].contains(selection) ? Angle(degrees: 0) : Angle(degrees: 180))
```

Comment:
I think it would be best to define the options in an array somewhere and point to that array here as well as below in line 54

Reply:
done